src/main/java/no/entur/uttu/stopplace/filter/BoundingBoxStopPlaceFilter.java

```diff
@@ -0,0 +1,3 @@
+package no.entur.uttu.stopplace.filter;
```

Comment:
You can leave this out if you're not planning to implement

Reply:
Will get it aways 👍